ddl/db_test.go

```diff
@@ -2725,18 +2723,18 @@ func (s *testDBSuite1) TestModifyColumnNullToNotNull(c *C) {
 		}
 		c2 := getModifyColumn()
 		if mysql.HasPreventNullInsertFlag(c2.Flag) {
```

Comment:
Do we need this check? In this line, I think `c2.Flag` is always `HasPreventNullInsertFlag` in this place.

Reply:
You're right, it not necessary.